src/views/Project/Details/Connector/ConnectorDetails/ConnectorSourceDetail/index.tsx

```diff
@@ -8,18 +8,21 @@ import {
     Area,
 } from 'recharts';
 
+import Message from '#rscv/Message';
 import TextOutput from '#components/general/TextOutput';
 
-import { PublishedDateCount } from '#typings';
+import {
+    ConnectorSourceStats,
+    ConnectorSourceStatus,
+} from '#typings';
 
 import styles from './styles.scss';
 
 interface OwnProps {
     title: string;
-    noOfLeads: number;
     className?: string;
-    broken?: boolean;
-    publishedDates?: PublishedDateCount[];
+    stats: ConnectorSourceStats;
```

Comment:
I think we should stop using this shortcut for `statistics`
Get's really confusing with `status`